Header header

string name
string status
string mode
uint32 number_of_clovers
uint32[] clover_ids